app/boards/intel_adsp_cavs25_tgph.conf

```diff
@@ -6,6 +6,9 @@ CONFIG_INTEL_SSP=y
 CONFIG_INTEL_ALH=y
 CONFIG_LP_MEMORY_BANKS=1
 CONFIG_HP_MEMORY_BANKS=30
+CONFIG_MM_DRV=y
+CONFIG_INTEL_MODULES=y
```

Comment:
The MM_DRV is compile time dependency (as per https://github.com/thesofproject/sof/pull/7986/commits/14e4b6c1bd869965afa15c9293b4d8a31044f547), I can drop the INTEL_MODULES as the code will compile now (https://github.com/thesofproject/sof/pull/7986/commits/78eeaa0c50b50dc988f37e7ba73552331659475a)

Reply:
> memory manager and tlb was prepared for mtl. Are you sure that it is valid configuration? @jxstelter we've talk about it ealier, any thoughts?

For library loading the generic MM driver will work. However instantiating library will fail since memory remapping is required. Currently only MTL specific code (mm_drv_intel_adsp_mtl_tlb.c) has remapping implemented.

Reply:
I guess it is fine if instantiating fails, but the loading works. For now. If we keep this part working than 'only' the instantiation needs to be worked on for TGL, if we decide to do so.

Reply:
ok, let revisit the MM if needed later. We can merge for now.